internal/resources/cluster/resource_cluster.go

```diff
@@ -346,12 +358,25 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, m interf
 			Summary:  "Kubernetes cluster's kubeconfig provided. Proceeding to attach the cluster TMC",
 		})
 
-		deploymentManifests, err := manifest.GetK8sManifest(clusterResponse.Cluster.Status.InstallerLink)
+		if clusterResponse.Cluster.Spec.ImageRegistry != "" {
```

Comment:
Isn't this same for proxy config also an don't just for image registry?

Reply:
Proxy as well, made the change